app/src/main/java/com/team41/boromi/auth/LoginFragment.java

```diff
@@ -14,6 +16,7 @@
 
 import com.team41.boromi.BookActivity;
 import com.team41.boromi.R;
+import org.w3c.dom.Text;
```

Comment:
what is with you importing weird libraries, what does this one do

Reply:
Make sure to optimize imports before you PR incasee you add an unused library when coding

Reply:
yes done